src/lib.rs

```diff
@@ -110,9 +112,14 @@ impl Platform for Std {
     }
 }
 
+#[cfg(feature="std")]
 #[derive(Debug, Clone, Eq, Hash, PartialEq)]
 pub struct SockAddr(libc::sockaddr_vm);
 
+#[cfg(not(feature="std"))]
+#[derive(Clone)]
+pub struct SockAddr(libc::sockaddr_vm);
```

Comment:
I think the duplicated definition of `SockAddr` could be avoided by writing:
```
#[derive(Clone)]
#[cfg_attr(feature = "std", derive(Debug, Eq, Hash, PartialEq))]
pub struct SockAddr(libc::sockaddr_vm);
```

Reply:
Oh good to know!